lib/api/exception.dart

```diff
@@ -11,13 +13,26 @@ sealed class ApiRequestException implements Exception {
   /// and its route name would be "getMessages".
   final String routeName;
 
-  /// A user-facing description of the error.
+  /// Server supplied user-facing description of the error.
+  ///
+  /// For [ZulipApiException] this may be supplied by the server as the `message`
+  /// property in the JSON response.
+  // TODO(i18n): no method yet to translate a server provided string.
+  final String? message;
```

Comment:
As the meaning of this is changing to be more specific, it should get a more specific name to match. `serverMessage` would fit this description well.